ledger/stateproofverificationtracker.go

```diff
@@ -118,40 +118,40 @@ func (spt *stateProofVerificationTracker) produceCommittingTask(_ basics.Round,
 }
 
 func (spt *stateProofVerificationTracker) prepareCommit(dcc *deferredCommitContext) error {
-	spt.stateProofVerificationMu.RLock()
-	defer spt.stateProofVerificationMu.RUnlock()
+	spt.mu.RLock()
+	defer spt.mu.RUnlock()
 
-	lastDataToCommitIndex := spt.committedRoundToLatestCommitDataIndex(dcc.newBase)
-	dcc.stateProofVerificationCommitData = make([]verificationCommitData, lastDataToCommitIndex+1)
-	copy(dcc.stateProofVerificationCommitData, spt.trackedCommitData[:lastDataToCommitIndex+1])
+	lastContextToCommitIndex := spt.roundToLatestCommitContextIndex(dcc.newBase)
+	dcc.spVerification.CommitContext = make([]verificationCommitContext, lastContextToCommitIndex+1)
+	copy(dcc.spVerification.CommitContext, spt.trackedCommitContext[:lastContextToCommitIndex+1])
 
-	dcc.stateProofVerificationLatestDeleteDataIndex = spt.committedRoundToLatestDeleteDataIndex(dcc.newBase)
-	if dcc.stateProofVerificationLatestDeleteDataIndex > -1 {
-		dcc.stateProofVerificationEarliestTrackStateProofRound = spt.trackedDeleteData[dcc.stateProofVerificationLatestDeleteDataIndex].stateProofNextRound
+	dcc.spVerification.LatestUsedDeleteContextIndex = spt.roundToLatestDeleteContextIndex(dcc.newBase)
+	if dcc.spVerification.LatestUsedDeleteContextIndex >= 0 {
+		dcc.spVerification.EarliestTrackLastAttestedRound = spt.trackedDeleteContext[dcc.spVerification.LatestUsedDeleteContextIndex].stateProofNextRound
```

Comment:
I still think these names should be shorter

Reply:
Shortened some more.